ACE/ace/Global_Macros.h

```diff
@@ -61,24 +61,11 @@
 #  define ACE_ENDLESS_LOOP
 # endif /* ! ACE_ENDLESS_LOOP */
 
-# if defined (ACE_NEEDS_FUNC_DEFINITIONS) && !defined (ACE_HAS_CPP11)
-    // It just evaporated ;-)  Not pleasant.
-#   define ACE_UNIMPLEMENTED_FUNC(f)
-# else
-#   if defined (ACE_HAS_CPP11)
-#     define ACE_UNIMPLEMENTED_FUNC(f) f = delete;
-#   else
-#     define ACE_UNIMPLEMENTED_FUNC(f) f;
-#   endif
-# endif /* ACE_NEEDS_FUNC_DEFINITIONS */
+# define ACE_UNIMPLEMENTED_FUNC(f) f = delete;
```

Comment:
I'm guessing we might want to keep macros like these for compatibility for users, but maybe we should replace these in ACE and TAO for more idiomatic code.

Reply:
If that's already the plan you can ignore me.

Reply:
Yes, we keep them for the users but want to replace this globally in ACE/TAO, a lot of cleanup can be done, help is welcome ;-)